src/main.rs

```diff
@@ -60,7 +71,17 @@ fn build_subcommands<'a, 'b>(
         }
         cli_app = cli_app.subcommand(subcmd);
     }
-    cli_app
+
+    // This is needed to prevent clap from complaining. It should be removed once
+    // clap 3.x is released. See https://github.com/clap-rs/clap/issues/748
+    let maskfile = Arg::with_name("maskfile")
+        .help("Points to a maskfile to use")
```

Comment:
I think specifying this is a "path" might be a little more explicit :)

```suggestion
        .help("Path to a different maskfile you want to use")
```